### ISSUE TITLE
Replace usage of realpath for maintaining osx compatibility

### ISSUE THREAD

**Issue by ageekymonk** [kubernetes-incubator/service-catalog#2200]
# Bug Report 

**What happened**:
In OSX, realpath is not part of the standard installation. Due to this we are not able to create a brew formula for osx. https://github.com/Homebrew/homebrew-core/pull/29646

**What you expected to happen**:
Replace realpath with bash string manipulation or something similar

**Comment by MHBauer** [comment]
In the linked PR, not sure why you want to run `make .generated_files` at all. Everything generated should be checked in.
/assign @carolynvs

**Comment by carolynvs** [comment]
@MHBauer Thanks for looping me in! I've worked with homebrew a bunch before on similar stuff so I can help shepherd getting their PR merged over there.

As you pointed out, our generated files are checked in so this issue can be closed. @ageekymonk I'll work with you on the homebrew PR to unblock you on whatever you need in order to get your homebrew PR merged.

**Comment by MHBauer** [comment]
Is the brew formula something we (service-catalog project) should own?

**Comment by carolynvs** [comment]
Homebrew encourages the community to support and maintain their formulas. Having the formula come from the community (and I checked it out, it follows their template for Go pretty much to a T), is a good thing. 💖
